package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "code-oss-dev",
-  "version": "1.55.0",
```

Comment:
we want to remove this change.

Reply:
good catch thanks - happened when I changed the base branch to release/1.55